<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>235</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>235</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filedir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="choiceButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Iconv GUI 0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="inputEncoding">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="outputEncoding">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="toLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>to</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveOldCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save old</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="useDirCheckBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Use dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="typesList">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>txt, srt</string>
    </property>
   </widget>
   <widget class="QLabel" name="usingTypesLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Using types:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
